sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemClient.java

```diff
@@ -511,6 +511,45 @@ public Response<FileSystemProperties> getPropertiesWithResponse(String leaseId,
         }, LOGGER);
     }
 
+    /**
+     * Determines if the file system exists in the cloud.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <!-- src_embed com.azure.storage.file.datalake.DataLakeFileSystemClient.exists -->
+     * <pre>
+     * System.out.printf&#40;&quot;Exists? %b%n&quot;, client.exists&#40;&#41;&#41;;
+     * </pre>
+     * <!-- end com.azure.storage.file.datalake.DataLakeFileSystemClient.exists -->
+     *
+     * @return true if the path exists, false if it doesn't
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Boolean exists() {
```

Comment:
Let's make this `boolean`, only reason it needed to be `Boolean` in the other overloads is because `boolean` isn't allowed in generics